Cubical/Categories/Displayed/DisplayOverProjections.agda

```diff
@@ -0,0 +1,131 @@
+{-# OPTIONS --safe #-}
+--
+module Cubical.Categories.Displayed.DisplayOverProjections where
```

Comment:
I don't have any good ideas for naming this module. 

I had this in `Displayed.Base` but it got too bloated. A rough first naming was `DisplayOverProjections` because we turn a display over `C ×C D` into a display over `C` or `D`, but I don't think this is a good module name.

Reply:
Maybe more principled naming for the terms here, instead of things like `mk∫ᴰsrFunctorᴰ`

Reply:
Good idea! We'll try it out